doc/tutorials/run_all_tests_with_docker.md

```diff
@@ -0,0 +1,133 @@
+# Introduction
+
+Running all the tests like the build server requires to have multiple dependencies installed. To overcome this problem, instead of trying to install all the necessary dependencies on your own, an appropriate Docker image can be used. This way you can easily and quickly run all the tests.
+
+## Who is this guide for?
```

Comment:
```suggestion
## Who Is This Guide For?
```

Please use [title case](https://titlecase.com) for headings as described [here](http://master.libelektra.org/doc/CODING.md#markdown-guidelines).